libparsec/crates/platform_mountpoint/src/memfs/mod.rs

```diff
@@ -0,0 +1,439 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
```

Comment:
We don't need this here (it should be in winfsp-wrs as example instead)

On top of that this uses the `MountpointInterface` which is no longer needed, so better to remove this code entirely than try to fix it !